scripts/common/one-inch.ts

```
@@ -2,7 +2,7 @@ import axios from 'axios'
 import BigNumber from 'bignumber.js'
 import { OneInchQuoteResponse, OneInchSwapResponse } from './types'
 
-const API_ENDPOINT = `https://oasis.api.enterprise.1inch.exchange/v4.0/1`
+const API_ENDPOINT = process.env.ONE_INCH_API_ENDPOINT || `https://api-oasis.1inch.io/v4.0/1`
```

instead of adding fallback which is just a redundancy (same as in config) better to throw exception when env is not available
this way you know immediately if config is wrong and make you on track to fix the issue